apps/examples/st_things/sample_light_things.c

```diff
@@ -0,0 +1,140 @@
+/* ****************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <signal.h>
+#include <stdbool.h>
+#include <st_things/st_things.h>
+#include <iotbus/iotbus_gpio.h>
+
+#define TAG		"LIGHT_THINGS"
+#define SW703_BUTTON	44
```

Comment:
Could you explain what it is? Is it a button of ARTIK053? In your Kconfig, there is no dependancy with ARTIK053. Is this an example only for ARTIK053?

Reply:
@KimDongEon

Reply:
modified

Reply:
Allow users to configure in the configuration.
1. Whether to reset the connection using the button
2. If 1 is set to Yes, the pin number can be set.

It no longer depends on ARTIK053.